week-21-Events/counter.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Workshop: EventListeners</title>
+</head>
+
+<body>
+  <ul>
+    <li>apple</li>
+    <li>balloon</li>
+    <li>cat</li>
+    <li>dolphin</li>
+  </ul>
+  <button>How many items are in the list?</button>
+  <p class="result">dunno :(</p>
+
+  <script>
+    // On the click of the button,
+    // Count the items in the list
+    // And display the result in the result element.
+    window.addEventListener('DOMContentLoaded', () => {
+      const button = document.querySelector('button');
+      const list = document.querySelector('ul');
+      const result = document.querySelector('.result');
+
+      const countItems = () => {
```

Comment:
Érdemes a `list` és a `result` változókat is a `countItems` függvényen belül létrehozni. Mindig van rá esély, hogy mire az event handler meghívódik, a korábban lekérdezett elem már nincs a DOM-ban.